config/custom-environment-variables.json

```diff
@@ -56,7 +56,8 @@
         "serviceApi": "SERVICES_HEARINGS_COMPONENT_API_ET",
         "caseTypes": "SERVICES_HEARINGS_CASETYPES_ET"
       },
-      "hmcApi": "SERVICES_HMC_HEARINGS_COMPONENT_API"
+      "hmcApi": "SERVICES_HMC_HEARINGS_COMPONENT_API",
+      "enablePreviewCcd": "ENABLE_HMC_PREVIEW_CCD"
```

Comment:
Naming things is one of the hard problems, but can we call this something else, because it's not specific to preview, hearings or CCD. I'd suggest enableDataSourceHeaders, with corresponding changes in the environment variable name too